hbase-client/src/main/java/org/apache/hadoop/hbase/client/RegionInfo.java

```diff
@@ -622,9 +630,9 @@ static RegionInfo createMobRegionInfo(TableName tableName) {
 
     int endOffset = regionName.length;
     // check whether regionName contains encodedName
-    if (regionName.length > MD5_HEX_LENGTH + 2
-    && regionName[regionName.length-1] == ENC_SEPARATOR
-    && regionName[regionName.length-MD5_HEX_LENGTH-2] == ENC_SEPARATOR) {
+    if (regionName.length > MD5_HEX_LENGTH + 2 &&
+        regionName[regionName.length-1] == ENC_SEPARATOR &&
+        regionName[regionName.length-MD5_HEX_LENGTH-2] == ENC_SEPARATOR) {
```

Comment:
We need this bit?

Reply:
Sure. Ensures delimiting periods. It was what was there before. Just fixed offset.